.github/scripts/action.yaml

```diff
@@ -0,0 +1,11 @@
+name: Copy library generation script
+description: Copy library generation shell script to the calling workflow
+
+runs:
+  using: "composite"
```

Comment:
I didn't use reusable workflow because, even though the reusable workflow defined here, the context of the runner is associated with the caller workflow.

When a reusable workflow is triggered by a caller workflow, the `github` context is always associated with the caller workflow. See https://docs.github.com/en/actions/sharing-automations/reusing-workflows#overview

Reply:
So looks like we are just reusing the scripts not the workflow? If that's the case, maybe we just bake the scripts into the image?